.github/workflows/main.yml

```diff
@@ -47,10 +51,28 @@ jobs:
 
     - uses: codecov/codecov-action@v1
 
-    - name: Build
+    # - name: Build
+    #   run: |
+    #     python setup.py sdist bdist_wheel
+
+
+
+
+  deploy:
+    needs: test
+    runs-on: ubuntu-latest
+    name: deploy
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Python '3.8'
+      uses: actions/setup-python@v2
+      with:
+        python-version: '3.8'
+    - name: "Install"
       run: |
+        python -m pip install --upgrade pip;
+        python -m pip install build
```

Comment:
The `pip install setuptools wheel twine` line should be removed above and added here